CMakeLists.txt

```diff
@@ -155,9 +155,7 @@ include_directories(${CMAKE_SOURCE_DIR}/third-party/include)
 install_uWebSockets()
 
 #pugixml
-SET(PUGIXML_COMPACT ON CACHE BOOL "" FORCE)
```

Comment:
We still need to set this option by default. I think that the same configuration line will work, but please test this.

Reply:
I'm sorry, maybe I got files mixed up here! 
I thought I had added  `-DPUGIXML_COMPACT` to line 251
```
add_definitions(-DHAVE_HDF5 -DPUGIXML_COMPACT)
```
I better wait until the morning.

Reply:
Oh yes, I must have removed it when testing scenario 2, and then made a different commit and didn't add it back.
I believe `-DPUGIXML_COMPACT` does the same as `SET(PUGIXML_COMPACT ON CACHE BOOL "" FORCE)`, in fact, I think it is the way we need to do it now.

Reply:
I'm sorry, now I made another mistake and added both those options in!
I definitely better wait until tomorrow before I make any more mistakes. Then I'll do a performance test on both scenarios.